build/configs/rtl8730e/audio/defconfig

```diff
@@ -463,7 +464,7 @@ CONFIG_I2C_POLLED=y
 # CONFIG_I2C_TRACE is not set
 # CONFIG_I2C_WRITEREAD is not set
 # CONFIG_SPI is not set
-CONFIG_GPIO=y
+# CONFIG_GPIO is not set
```

Comment:
it should be enabled to handle wake up

Reply:
Audio defconfig diverged in between and I am not sure of which defconfig was used as reference for creating it.
So I am not sure about some other configs too in audio.
Please give me some more info and then I can modify audio defconfig as per latest requirements.

Reply:
except this config, rest all changes wont affect audio related code. So, please revert CONFIG_GPIO change.